README.md

```diff
@@ -1,4 +1,750 @@
+		if response.StatusCode != http.StatusOK {
+			return fmt.Errorf("Error Listing Topics: status %d\n", response.StatusCode)
+		}
+	
+		// Loop and print topics.
+		for _, topicDetail := range result {
+			fmt.Printf("\tname: %s\n", *topicDetail.Name)
+		}
+		return nil
+	} // func.end
+
+
+
+### Getting a Kafka topic
+---
+To get a Kafka topic detail information, issue a GET request to the `/admin/topics/TOPICNAME`
+path (where `TOPICNAME` is the name of the Kafka topic that you want to get).  
+
+Expected status codes
+  - 200: Retrieve topic details successfully in following format:
+```json
+{
+  "name": "MYTOPIC",
+  "partitions": 1,
+  "replicationFactor": 3,
+  "retentionMs": 86400000,
+  "cleanupPolicy": "delete",
+  "configs": {
+    "cleanup.policy": "delete",
+    "min.insync.replicas": "2",
+    "retention.bytes": "1073741824",
+    "retention.ms": "86400000",
+    "segment.bytes": "536870912"
+  },
+  "replicaAssignments": [
+    {
+      "id": 0,
+      "brokers": {
+        "replicas": [
+          3,
+          2,
+          4
+        ]
+      }
+    }
+  ]
+}
+```
+
+#### Example
+	func topicDetails(serviceAPI *adminrestv1.AdminrestV1) error {
+		// Construct an instance of the GetTopicOptions model
+		getTopicOptionsModel := new(adminrestv1.GetTopicOptions)
+		getTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+		getTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+	
+		// Call List Topic Details.
+		result, response, operationErr := serviceAPI.GetTopic(getTopicOptionsModel)
+		if operationErr != nil {
+			return fmt.Errorf("Error Listing Topic Details" + operationErr.Error())
+		}
+	
+		// Check the result.
+		if response.StatusCode != http.StatusOK {
+			return fmt.Errorf("Error Listing Topic Details: status %d\n", response.StatusCode)
+		}
+	
+		// Print topics details.
+		fmt.Printf("\tname: \t\t\t%s\n", *result.Name)
+	
+		// The number of partitions.
+		fmt.Printf("\tno of partitions: \t%d\n", *result.Partitions)
+	
+		// The number of replication factor.
+		fmt.Printf("\treplication factor: \t%d\n", *result.ReplicationFactor)
+	
+		// // The value of config property 'retention.ms'.
+		fmt.Printf("\tretention (ms): \t%d\n", *result.RetentionMs)
+	
+		// // The value of config property 'cleanup.policy'.
+		fmt.Printf("\tcleanup policy: \t%s\n", *result.CleanupPolicy)
+	
+		// Configs *TopicConfigs
+		fmt.Printf("\ttopic configs: \t\t%+v\n", *result.Configs)
+	
+		// The replia assignment of the topic.
+		// ReplicaAssignments []ReplicaAssignment
+		for _, assignment := range result.ReplicaAssignments {
+			fmt.Printf("\tassignment:  \t\tid:%d,  \tbrokers: %+v\n", assignment.ID, assignment.Brokers)
+		}
+		return nil
+	} // func.end
+
+
+
+### Updating Kafka topic's configuration
+---
+To increase a topic's partition number or to update a topic's configuration, the admin REST SDK issues an
+`PATCH` request to `/admin/topics/{topic}` with the following body:
```

Comment:
In other section, you used something like

> To get a Kafka topic detail information, issue a GET request to the `/admin/topics/TOPICNAME`
path (where `TOPICNAME` is the name of the Kafka topic that you want to get).  

Shall we keep it consistent?

Reply:
To increase a Kafka topic's partition number or to update a Kafka topic's configuration, issue a
`PATCH` request to `/admin/topics/TOPICNAME` with the following body:
(where TOPICNAME is the name of the Kafka topic that you want to update).